meshtastic/mesh_interface.py

```
@@ -81,13 +83,62 @@
+        current_logger = current_logger.parent
+    return False
+
+
+def _emit_response_summary(message: str) -> None:
+    """Emit a short response summary without hiding legacy stdout behavior."""
+    logger.info("%s", message)
+    if not _logger_has_visible_info_handler(logger):
+        print(message)
+
+
+def _normalize_json_serializable(value: Any) -> Any:
```

Ruff (0.15.4)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)